ci/scripts/execute_tests.sh

```diff
@@ -88,14 +93,6 @@ GRADLE_ARGS="-PtestJVM=${JAVA_TEST_PATH} \
     ${GRADLE_TASK} \
     ${GRADLE_TASK_OPTIONS}"
 
-case $ARTIFACT_SLUG in
-  windows*)
-    EXEC_COMMAND="bash -c 'export JAVA_HOME=${JAVA_BUILD_PATH}; echo Building with:; "'"'"${JAVA_BUILD_PATH}\bin\java.exe"'"'" -version; echo Testing with:; "'"'"${JAVA_TEST_PATH}\bin\java.exe"'"'" -version; cd geode; ./gradlew ${GRADLE_ARGS}'"
-    ;;
-  *)
-    EXEC_COMMAND="bash -c 'export JAVA_HOME=${JAVA_BUILD_PATH} && echo Building with: && ${JAVA_BUILD_PATH}/bin/java -version && echo Testing with: && ${JAVA_TEST_PATH}/bin/java -version && cd geode && ./gradlew ${GRADLE_ARGS}'"
-    ;;
-esac
-
+EXEC_COMMAND="bash -c 'echo Building with: $SEP ${JAVA_BUILD_PATH}/bin/java -version $SEP echo Testing with: $SEP ${JAVA_TEST_PATH}/bin/java -version $SEP cd geode $SEP sed -e 's/JAVA_HOME/GRADLE_JVM/g' -i.bak gradlew $SEP GRADLE_JVM=${JAVA_BUILD_PATH} ./gradlew ${GRADLE_ARGS}'"
```

Comment:
I dislike this a great deal less than actually modifying the gradlew.  Woot woot!